docs/source/documentation/diagrams/architecture-diagrams.html.md.erb

```diff
@@ -0,0 +1,33 @@
+# Data Platform - Architectural Diagrams
+
+## C4 Model Diagrams
+
+In the [C4 model](https://c4model.com/), the four C's represent [context](https://c4model.com/#SystemContextDiagram), [containers](https://c4model.com/#ContainerDiagram), [components](https://c4model.com/#ComponentDiagram), and [code](https://c4model.com/#CodeDiagram). These different levels help you to tell the right story to the right audience.
+
+### Level 1 - Context Diagram
+This is the highest level of abstraction and provides an overview of the entire system. It shows the system under consideration and its external dependencies, such as users, external systems, and APIs. Frequency in change will be minimal compared to other levels.
+
+![Level 1 Diagram](c4-model-level1.excalidraw.png)
+
+### Level 2 - Container Diagram
+
+This level focuses on the software system's high-level [containers](https://c4model.com/#ContainerDiagram), which are typically applications or services. It illustrates the major components of the system and their interactions. Containers can be web servers, databases, desktop applications, mobile apps, etc.
+
+![Level 2 Diagram](c4-model-level2.excalidraw.png)
+
+### Level 3 - Component
+
+This level delves deeper into the containers identified in the previous level. It shows the internal components and their interactions within each container. Components can represent classes, modules, services, or other logical units.
+
+####  3.1 Data Product Component - Proposal
+
+This diagram shows the relationships of a Data Product with other components of the Data Platform.
+
+![Level 3 Diagram](c4-model-level3-data-product.excalidraw.png)
+
+## Editing the diagrams
+
+These diagrams were created using the open source tool [Excalidraw](https://docs.excalidraw.com/). The source of the excalidraw files are embedded into the `.png` format creating a `.excalidraw.png` file.
+
+- On the web using [vscode.dev](https://github.dev/ministryofjustice/data-platform/tree/main/docs/diagrams) with the [Excalidraw extenstion](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor)
```

Comment:
It looks like this bullet point and the next one need an intro. Maybe just something as simple as 'There are multiple ways you can use Excalidraw:'